Core/Utilities/File.py

```diff
@@ -4,25 +4,35 @@
    By default on Error they return None.
 """
 
-__RCSID__ = "$Id$"
-
 import os
 import hashlib as md5
 import random
 import glob
-import types
+import sys
 import re
+import errno
+
+__RCSID__ = "$Id$"
+
+def mkDir( path ):
+  try:
```

Comment:
No I don't. Other exceptions are not supposed to happen: if they do, it's a problem.